src/index.html

```diff
@@ -15,20 +26,36 @@ <h1>Excepteur Sint Occaecat</h1>
         <a href="#">second point</a>
       </li>
       <li class="">
-        <a href="#"> <!-- use .list-item-link class if you are confused about nested selectors -->
+        <a href="#">
+          <!-- use .list-item-link class if you are confused about nested selectors -->
           third point
         </a>
       </li>
     </ul>
     <h2 class="highlight point">Duis Aute</h2>
-    <p>Quis nostrud exercitation ullamco <a href="https://yahoo.com" target="_blank">laboris</a> nisi ut aliquip ex ea commodo consequat.</p>
-    <img src="https://www.rd.com/wp-content/uploads/2016/01/04-dog-breeds-dalmation.jpg" width="150" alt="a Dalmatian dog">
-    <audio src="https://github.com/nbktechworld/introduction-to-html/raw/abcdac58651ba724c40665c82dd55f171534115f/document-two-sections/barking.mp3" controls></audio>
-    <ol>
+    <p>
+      Quis nostrud exercitation ullamco
+      <a href="https://yahoo.com" target="_blank">laboris</a>
+      nisi ut aliquip ex ea commodo consequat.
+    </p>
+    <img
+      src="https://www.rd.com/wp-content/uploads/2016/01/04-dog-breeds-dalmation.jpg"
+      width="150"
+      alt="a Dalmatian dog"
+    />
+    <audio
+      src="https://github.com/nbktechworld/introduction-to-html/raw/abcdac58651ba724c40665c82dd55f171534115f/document-two-sections/barking.mp3"
+      controls
+    ></audio>
+    <ol class="order">
       <li>first point</li>
       <li>second point</li>
       <li>third point</li>
     </ol>
-    <video src="http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/BigBuckBunny.mp4" controls width="300"></video>
+    <video
```

Comment:
Try to not change code unrelated to the issue at hand.

You should focus on the issue and what it's asking. If the issue doesnt require a code change here, you shouldnt be changing it. Even if you think the style is prettier the way you write here. You should strive to do the least amount of work as possible and avoid "diff noise" by changing unrelated code just because you feel a certain way about how it looks.